.golangci.yml

```diff
@@ -14,11 +14,14 @@ linters-settings:
       - NOCOMMIT
   govet:
     disable:
+      - assign
+      - atomic
       - copylocks
 
 linters:
   enable-all: true
   disable:
+    - bodyclose
```

Comment:
why are we disabling so may linters, can we not fix them ?

Reply:
These all are the new linters added after 1.26.0 and on or before 1.50.1 release. Golang 19 won't support the linter of version 1.26.0. So for now we disabled some.